README.rst

```diff
@@ -13,6 +13,9 @@ Wger-kronos web application
 .. image:: https://api.codacy.com/project/badge/Grade/e7b7ec017fde4621be10eaeae244876f
     :target: https://www.codacy.com/app/Emmanuel-Dominic/wger-kronos?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=andela/wger-kronos&amp;utm_campaign=Badge_Grade
 
+.. image:: https://coveralls.io/repos/github/andela/wger-kronos/badge.svg?branch=develop
```

Comment:
@reifred Awesome add of coverage badge. It was reflecting an unknown on the branch `README`  but now I can see that it will definitely reflect the coverage on our default branch.